cmd/svcat/testdata/plugin.yaml

```diff
@@ -88,13 +88,26 @@ tree:
     - name: all-namespaces
       desc: If present, list the requested object(s) across all namespaces. Namespace
         in current context is ignored even if specified with --namespace
+    - name: output
```

Comment:
Oh gosh, I just realized that the doc I added to the dev guide explaining how to easily update the golden files went missing! 😝 I hope that didn't waste too much of your time.

The trick is this (i'll submit a PR in a minute to fix the doc)

1. touch cmd/svcat/testdata/output/NEWFILE
2. go test ./cmd/svcat/... -update

That will insert the test output into the golden file without you having to do it manually.